### PR TITLE
adding in moment.js and fomatting due at time

### PR DESCRIPTION
**Pull request by jivewise**
screen shot 2015-02-20 at 4 03 16 pm

Adding in moment.js and formatting due_at timestamp.  Is there a better way of doing this than in the view directly?

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "jquery": "~2.1.1",
     "bootstrap": "~3.2.0",
     "katex": "~0.1.1",
-    "bootstrap-material-design": "~0.2.2"
+    "bootstrap-material-design": "~0.2.2",
+    "moment": "~2.9.0"
```
Comment (philschatz) [inline bower.json]: Is the bower `moment.js` used anywhere?

File: src/components/task.cjsx [hunk 2]
```diff
@@ -105,7 +105,7 @@ module.exports = React.createClass
           {breadcrumbs}
           <BS.Panel bsStyle="default" footer={footer}>
             <h1>{model.title}</h1>
-            <p>Due At: {model.due_at}</p>
+            <p>Due At: {moment(model.due_at).format('LLL')}</p>
```
Comment (philschatz) [inline src/components/task.cjsx]: Should we have a `<Time>` component that generates a `<time>` HTML element? That way date/time formatting is done consistently throughout the app?

Comment (jivewise) [inline src/components/task.cjsx]: Sure, I can do that.

File: bower.json (hunk 1 above)
Comment (jivewise) [inline bower.json]: Nope I'll remove it.

**Pull request merged**